searchcommon/src/vespa/searchcommon/common/schema.cpp

```diff
@@ -131,60 +131,48 @@ Schema::Field::operator!=(const Field &rhs) const
 
 Schema::IndexField::IndexField(vespalib::stringref name, DataType dt)
     : Field(name, dt),
-      _prefix(false),
-      _phrases(false),
-      _positions(true),
-      _avgElemLen(512)
+      _avgElemLen(512),
+      _experimental_posting_list_format(false)
 {
 }
 
 Schema::IndexField::IndexField(vespalib::stringref name, DataType dt,
                                CollectionType ct)
     : Field(name, dt, ct),
-      _prefix(false),
-      _phrases(false),
-      _positions(true),
-      _avgElemLen(512)
+      _avgElemLen(512),
+      _experimental_posting_list_format(false)
 {
 }
 
 Schema::IndexField::IndexField(const std::vector<vespalib::string> &lines)
     : Field(lines),
-      _prefix(ConfigParser::parse<bool>("prefix", lines)),
-      _phrases(ConfigParser::parse<bool>("phrases", lines)),
-      _positions(ConfigParser::parse<bool>("positions", lines)),
-      _avgElemLen(ConfigParser::parse<int32_t>("averageelementlen", lines))
+      _avgElemLen(ConfigParser::parse<int32_t>("averageelementlen", lines)),
+      _experimental_posting_list_format(ConfigParser::parse<bool>("experimentalpostinglistformat", lines))
 {
 }
 
 void
 Schema::IndexField::write(vespalib::asciistream & os, vespalib::stringref prefix) const
 {
     Field::write(os, prefix);
-    os << prefix << "prefix " << (_prefix ? "true" : "false") << "\n";
-    os << prefix << "phrases " << (_phrases ? "true" : "false") << "\n";
-    os << prefix << "positions " << (_positions ? "true" : "false") << "\n";
```

Comment:
This will likely break downgrade due to loading schema from file not using default value.

Reply:
Fixed